themes/src/main/resources/theme/keycloak/login/resources/css/password-entropy.css

```
@@ -0,0 +1,8 @@
+#password-entropy {
+    display: none;
+}
+
+#password-entropy > .progress {
```

There should be no need to use a child combinator here, using the class name should be more than specific enough.
```suggestion
#password-entropy .progress {
```